agent/jvm/pom.xml

```diff
@@ -184,6 +208,30 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>testing-on-ibm-jvm</id>
+      <activation>
+        <property>
+          <name>java.vendor</name>
+          <value>IBM Corporation</value>
+        </property>
+      </activation>
+      <properties>
+        <jvm.algorithm>IBMX509</jvm.algorithm>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <forkMode>always</forkMode>
+              <argLine>-javaagent:${settings.localRepository}/org/jmockit/jmockit/1.8/jmockit-1.8.jar</argLine>
```

Comment:
would be cool when if we could reuse the same jmockit version as declared in the dependency. Maybe extract it into a property which then is referenced here and within the dependency ?

Reply:
Correct, my bad. I intended to change it before committing but this bad boy slipped through :)